omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/EncounterController1_9Test.java

```diff
@@ -500,6 +501,31 @@ public void shouldUnVoidAnEncounter() throws Exception {
 		
 	}
 	
+	@Test
+	public void shouldFetchAllPatientEncounterIncludingVoidedPatientEncountersIfIncludeAllParamIsSetToTrue()
+	        throws Exception {
+		String encounterUuuid = "c59b3942-4fdd-11e5-8c3c-410f8777c163";
```

Comment:
Do you intend to reassign values to encounterUuuid or patientUuid?

Reply:
not realy , let me make them constatnts

Reply:
corrected it